### PR TITLE
Initial packer support

### PR DESCRIPTION
**Pull request by jesseadams**

### CODE REVIEW

**Comment by jesseadams** [comment]
Tests pass: https://circleci.com/gh/stelligent/minimal-pipeline-gem/21?utm_campaign=chatroom-integration&utm_medium=referral&utm_source=slack

File: lib/minimal_pipeline/cloudformation.rb [hunk 1]
```diff
@@ -80,17 +80,17 @@ def stack_output(stack, output)
     # @param stack_parameters [Hash] Parameters to be passed into the stack
     def deploy_stack(stack_name, stack_parameters)
       unless @client.describe_stacks(stack_name: stack_name).stacks.empty?
-        puts 'Updating the existing stack'
+        puts 'Updating the existing stack' if ENV['DEBUG']
```
Comment (ChrisPetr0) [inline lib/minimal_pipeline/cloudformation.rb]: Are these really debug messages?   I like them all the time

File: lib/minimal_pipeline/packer.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    def initialize(packer_config)
+      @config = packer_config
+    end
+
+    # Parse the newly built AMI from a given packer command output
+    #
+    # @param output [String] The command output of a packer run
+    # @return [String]
+    def get_ami_id(output)
+      return if output.nil? || output.empty?
```
Comment (ChrisPetr0) [inline lib/minimal_pipeline/packer.rb]: should this throw an error?

File: lib/minimal_pipeline/cloudformation.rb (hunk 1 above)
Comment (jesseadams) [reply lib/minimal_pipeline/cloudformation.rb]: Then enable `DEBUG=true` in your pipelines. This is aimed to keep rspec test output clean.

File: lib/minimal_pipeline/packer.rb (hunk 2 above)
Comment (jesseadams) [reply lib/minimal_pipeline/packer.rb]: Perhaps. I voted to leave it to the consumer of the library

**Pull request merged**